frontends/web/src/routes/device/manage-backups/manage-backups.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ * Copyright 2024 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { TDevices } from '../../../api/devices';
+import { ButtonLink } from '../../../components/forms';
+import { Guide } from '../../../components/guide/guide';
+import { Entry } from '../../../components/guide/entry';
+import { Header } from '../../../components/layout';
+import { Backups } from '../bitbox01/backups';
+import { BackupsV2 } from '../bitbox02/backups';
+import { SDCardCheck } from '../bitbox02/sdcardcheck';
+
+type TProps = {
+  deviceID: string;
+  devices: TDevices
+}
+
+export const ManageBackups = ({
+  deviceID,
+  devices,
+}: TProps) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  if (!devices[deviceID]) {
+    navigate('/');
+    return null;
+  }
+
+  return (
+    <div className="contentWithGuide">
+      <div className="container">
+        <div className="innerContainer scrollableContainer">
+          <Header
+            title={<h2>{t('backup.title')}</h2>}
+          />
+          <div className="content padded">
+            <h3 className="subTitle">{t('backup.list')}</h3>
+            <BackupsList
+              deviceID={deviceID}
+              devices={devices}
+            />
+          </div>
+        </div>
+      </div>
+      <ManageBackupGuide
+        deviceID={deviceID}
+        devices={devices}
+      />
+    </div>
+  );
+};
+
+const BackupsList = ({
+  deviceID,
+  devices,
+}: TProps) => {
+  switch (devices[deviceID]) {
+  case 'bitbox':
+    return (
+      <Backups
+        deviceID={deviceID}
+        showCreate={true}
+        showRestore={false}>
+        <BackButton deviceID={deviceID} />
+      </Backups>
+    );
+  case 'bitbox02':
+    return (
+      <SDCardCheck deviceID={deviceID}>
+        <BackupsV2
+          deviceID={deviceID}
+          showCreate={true}
+          showRestore={false}
+          showRadio={false}>
+          <BackButton deviceID={deviceID} />
+        </BackupsV2>
+      </SDCardCheck>
+    );
+  default:
+    return;
```

Comment:
nit: this could be `return null` to express that this component will be rendered but nothing is aborted?

Reply:
fixed